code/game/objects/items/melee/f13onehanded.dm

```diff
@@ -780,14 +708,7 @@
 ///////////		-generally max 24 damage
 
 
-/obj/item/melee/onehanded/knife/kitchen
-	name = "kitchen knife"
-	desc = "Simple kitchen knife.."
-	icon_state = "knife_kitchen"
-	force = 15
-	armour_penetration = 0
-
-/obj/item/melee/onehanded/knife/butcher
+/obj/item/melee/onehanded/knife/butcher// obsolete
```

Comment:
```suggestion
// obsolete
/obj/item/melee/onehanded/knife/butcher
```
Careful where you put comments. This would be read as a continuation of the type path.

Reply:
Good point!
I would remove it instantly if it wouldn't cause conflicts due to "merge lag", using obsolete tags to mark stuff I cant remove immediatly is my way of trying to make cleanups from repathing a little atomized,